packages/react-cosmos-webpack/package.json

```diff
@@ -16,7 +16,8 @@
     "html-webpack-plugin": "^2.24.1",
     "json-loader": "^0.5.4",
     "loader-utils": "^0.2.16",
-    "react-cosmos": "^2.0.0-beta.2",
+    "react-cosmos": "^2.0.0-beta.3",
+    "react-cosmos-utils": "^2.0.0-beta.2",
```

Comment:
No need to update dep versions. Lerna takes care of them automagically when running the publish cmd.